src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![warn(missing_docs, clippy::default_numeric_fallback)]
```

Comment:
We will keep this one forever, it's a safeguard against errors. See #932 if curious.

Reply:
Ah sorry misread the warn as allow 😳